docs/component.md

```diff
@@ -211,6 +212,40 @@ This will result in the following html:
 <my-button>
 ```
 
+#### Booleans
+Passing in booleans is handled similarly, but instead of adding an string attribute you either add the attribute or not. So your initial element might look like this:
+```html
+<my-button foo-bool><my-button>
+```
+In this case foo-bool equals true, when not passing in the attribute at all it equals false.
+If you script looks like this...
+
+```js
+import Component from '@biotope/element';
+
+class MyButton extends Component {
+
+    static attributes = [{name: 'foo-bool', converter: () => true}];
```

Comment:
This is ts not js. Please use the same style as it's done with `componentName`

Reply:
I would like to add, can you try to remove any `foo` and `boo` just use stuff that may be possible attributes so for a button something like `primary` for example. It makes the docs more useful imho 👍

Reply:
Adjusted file according to both of your points. Everything fine now? 👀